modules/op-starting-a-task-run-pipeline-run-build-user.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/unprivileged-building-of-container-images-using-buildah.adoc
+:_content-type: PROCEDURE
+
+[id="starting-a-task-run-with-custom-config-map-or-a-pipeline-run_{context}"]
+= Starting a task run with custom config map, or a pipeline run
+
+After defining the custom Buildah cluster task, you can create a `TaskRun` object that builds an image as a `build` user with user id `1000`. In addition, you can integrate the `TaskRun` object as part of a `PipelineRun` object. 
+
+.Procedure
+
+. Create a `TaskRun` object with a custom `ConfigMap` and `Dockerfile` objects.
++
+.Example: A task run that runs Buildah as user id `1000`
+[source,yaml]
+----
+apiVersion: v1
+data:
+  Dockerfile: |
+    ARG BASE_IMG=registry.access.redhat.com/ubi8/ubi
+    FROM $BASE_IMG AS buildah-runner
+    RUN dnf -y update && \
+        dnf -y install git && \
+        dnf clean all
+    CMD git
+kind: ConfigMap
+metadata:
+  name: dockerfile <1>
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: buildah-as-user-1000
+spec:
+  serviceAccountName: pipelines-sa-userid-1000
+  params:
+  - name: IMAGE
+    value: image-registry.openshift-image-registry.svc:5000/test/buildahuser
+  taskRef:
+    kind: Task
+    name: buildah-as-user
+  workspaces:
+  - configMap:
+      name: dockerfile <2>
+    name: source
+----
+<1> You can use a config map because the focus is on the task run, without any prior task that fetches some sources with a Dockerfile. 
+<2> You can mount a config map as the source workspace for the `buildah-as-user` task.
+
+. (Optional) You can create a pipeline and a corresponding pipeline run.
```

Comment:
I think `(Optional)` implies "You can" 

So I would write it out as a command, `Create a pipeline...`

Reply:
Umm, the entire step is optional. So I think it's not only a question of capability illustrated by "can". It's better to explicitly mention it.